shell/bootstrap-ocaml.sh

```diff
@@ -12,16 +12,16 @@ else
 fi
 mkdir -p bootstrap
 cd bootstrap
-URL=`sed -ne 's/URL_ocaml *= *//p' ../src_ext/Makefile | tr -d '\r'`
-MD5=`sed -ne 's/MD5_ocaml *= *//p' ../src_ext/Makefile | tr -d '\r'`
-V=`echo ${URL}| sed -e 's|.*/\([^/]*\)\.tar\.gz|\1|'`
-FV_URL=`sed -ne 's/URL_flexdll *= *//p' ../src_ext/Makefile | tr -d '\r'`
-FLEXDLL=`echo ${FV_URL}| sed -e 's|.*/\([^/]*\)|\1|'`
+URL=$(sed -ne 's/URL_ocaml *= *//p' ../src_ext/Makefile | tr -d '\r')
+MD5=$(sed -ne 's/MD5_ocaml *= *//p' ../src_ext/Makefile | tr -d '\r')
+V=$(echo ${URL}| sed -e 's|.*/\([^/]*\)\.tar\.gz|\1|')
+FV_URL=$(sed -ne 's/URL_flexdll *= *//p' ../src_ext/Makefile | tr -d '\r')
+FLEXDLL=$(echo ${FV_URL}| sed -e 's|.*/\([^/]*\)|\1|')
```

Comment:
There is still the odd Solaris out there, and the default `sh` on still just-about support Solaris doesn't recognise `$()`. These need to stay as they are (the ones in the Windows-specific part don't matter).

Reply:
oh wow! I did not know that. They’re not using xpg4 or xpg6?

Reply:
I don't know the full detail - I just know that Solaris 10 uses an incredibly old `sh` by default (and IIRC a crazy `patch` too). I'm all for `$()` but for the most part, I tend to leave existing uses of backtick alone (you can still shoot yourself in the foot: cf. https://github.com/ocaml/opam/pull/3699#issuecomment-446939748)

Reply:
What I’m seeing is that Solaris provides multiples implementations of the tools, each following a standard.

> An application that wants to use standard-conforming utilities must set the PATH (sh(1) or ksh(1)) or path (csh(1)) environment variable to specify the directories listed below in the order specified to get the appropriate utilities:
>
> **POSIX.1–2001, SUSv3**
>
> - `/usr/xpg6/bin`
> - `/usr/xpg4/bin`
> - `/usr/bin`
> - directory containing binaries for your compiler
> - other directories containing binaries needed by the application

https://docs.oracle.com/cd/E86824_01/html/E54776/xpg-5.html

We *could* set the `PATH` to use the latest stuff :smile: